lib/sendWelcomeEvent.ts

```diff
@@ -29,7 +30,7 @@ export const sendWelcomeEventToDialogFlow = async (app: IApp, read: IRead,  modi
         const response: IDialogflowMessage = await Dialogflow.sendRequest(http, read, modify, rid, event, DialogflowRequestType.EVENT);
         await createDialogflowMessage(rid, read, modify, response, app);
     } catch (error) {
-        console.error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${error.message}`);
+        console.error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${getError(error).message}`);
```

Comment:
@ItzzRitik @Shailesh351 this implementation of 'getError' is not working yet.  The log point looks like this:

Reply:
@ear-dev https://github.com/WideChat/Apps.Dialogflow/pull/98#discussion_r705388569